zstacklib/zstacklib/gpu/vendors/amd.py

```
@@ -0,0 +1,123 @@
+            data = json.loads(output.strip())
+        except Exception as e:
+            logger.error("Failed to parse AMD metrics JSON: %s" % str(e))
+            return results
+            
+        for card_name, card_data in data.items():
+            try:
+                pci_bus = card_data.get('PCI Bus', '')
+                pci_address = cls.normalize_pci_address(pci_bus)
+                
+                util = card_data.get('GPU use (%)')
+                mem_util = card_data.get('VRAM Total Memory (B)') # Simplified
```

Ruff (0.14.13)

Local variable `mem_util` is assigned to but never used

Remove assignment to unused variable `mem_util`

(F841)